src/imex.rs

```diff
@@ -166,7 +171,7 @@ pub async fn has_backup_old(context: &Context, dir_name: impl AsRef<Path>) -> Re
             let name = name.to_string_lossy();
             if name.starts_with("delta-chat") && name.ends_with(".bak") {
                 let sql = Sql::new();
-                if sql.open(context, &path, true).await {
+                if sql.open(context, &path, true).await.is_ok() {
```

Comment:
Same as above, it is uncommon to have files named delta-chat....bak which can't be opened as SQLite, could log a warning in this case.